Source/WebCore/dom/Node.cpp

```diff
@@ -118,11 +118,13 @@ struct SameSizeAsNode : EventTarget, CanMakeCheckedPtr<SameSizeAsNode> {
 #endif
     uint32_t refCountAndParentBit;
     uint32_t nodeFlags;
+    uint16_t elementStateFlags;
+    uint16_t styleBitfields;
     void* parentNode;
     void* treeScope;
-    uint8_t previous[8];
+    void* previous;
```

Comment:
Does this have to be a `void*`?  Can it be `Node*`?

Reply:
This type is just a proxy for the layout of `Node` so we can `static_assert` that `Node` doesn't unexpectedly change in size. I'm just updating it to reflect the types a bit better, but there's no need for it to be more precise so long as the size and alignment of fields are the same.